app/models/concerns/foreman_bootdisk/host_ext.rb

```diff
@@ -33,11 +33,17 @@ def bootdisk?
     managed? && bootdisk_build? && SETTINGS[:unattended]
   end
 
+  def bootdisk_downloadable?
+    ['ia32', 'x64'].include?(architecture.try(:bootfilename_efi))
```

Comment:
@domcleal [`Architecture#bootfilename_efi`](https://github.com/theforeman/foreman/blob/develop/app/models/architecture.rb#L19-L30) gives a more refined result of the arch. Although I could have checked for '86' in `host.arch` but to avoid generic set 'ia32' and 'x64' looks to be more prominent architectures belonging to intel.